src/js/App/Header/LogoutAlert.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
+import cookie from 'js-cookie';
+
+class LogoutAlert extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            alertOneVisible: cookie.get('cs_loggedOut') === 'true',
+            delay: 5000
+        };
+    }
+
+    hideAlertOne = () => {
+        this.setState({ alertOneVisible: false });
+        cookie.set('cs_loggedOut', 'false');
+    }
+    componentDidMount() {
```

Comment:
There should also be `componentWillUnmount` where we'd call `clearTimeout(this._timer);` in order to properly clear all timers.

Reply:
done